File: Domain/RentACarService.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class RentACarService
    {
        public Guid RentACarServiceId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public float AverageGrade { get; set; }
        public int NumberOfGrades { get; set; }
        public string Address { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public decimal TotalProfit { get; set; }
        public ICollection<AppUser> Managers { get; set; }

        public ICollection<Vehicle> Vehicles { get; set; }
        public ICollection<BranchOffice> BranchOffices { get; set; }
    }
}

File: front-end/src/app/models/rentACarService.js

export class RentACarService {
    constructor(rentACarServiceId, name, description, averageGrade, numberOfGrades, address, state, city, totalProfit, managers, vehicles, branchOffices) {
        this.rentACarServiceId = rentACarServiceId
        this.name = name
        this.description = description
        this.averageGrade = averageGrade
        this.numberOfGrades = numberOfGrades
        this.address = address
        this.state = state
        this.city = city
        this.totalProfit = totalProfit
        this.managers = managers
        this.vehicles = vehicles
        this.branchOffices = branchOffices
    }
}